x11-packages/xorg-mkfontscale/build.sh

```diff
@@ -13,9 +13,10 @@ TERMUX_PKG_CONFLICTS="xorg-mkfontdir"
 TERMUX_PKG_REPLACES="xorg-mkfontdir"
 
 termux_step_create_debscripts() {
-	for i in $(test "$TERMUX_PACKAGE_FORMAT" != "pacman" && echo postinst) postrm triggers; do
+	for i in postinst postrm triggers; do
 		cp "${TERMUX_PKG_BUILDER_DIR}/${i}" ./${i}
 		sed -i "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" ./${i}
 	done
+	[[ "$TERMUX_PACKAGE_FORMAT" != "pacman" ]] || echo "post_install" > postupg
```

Comment:
As truboxl mentioned above.
Since this only applies to Pacman this should be a truthy check.
```suggestion
	[[ "$TERMUX_PACKAGE_FORMAT" == "pacman" ]] && echo "post_install" > postupg
```